roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -129,20 +129,22 @@ class TransitionSpec extends FlatSpec with Matchers {
   "Executing bytecode from expression \"(+ 1 (+ 2 3))\"" should "result in Fixnum(6)" in {
 
     /**
+      * NOTE that in Roscala we are using one-byte offset for every opcode,
```

Comment:
It applies to all tests in this test suite. So let's move this note to the beginning of the test suite instead of adding it to each test.